docker-app/worker_wrapper/wrapper.py

```diff
@@ -314,46 +323,45 @@ def before_docker_run(self) -> None:
         self.data_last_packaged_at = timezone.now()
 
     def after_docker_run(self) -> None:
-        # only successfully finished packaging jobs should update the Project.data_last_packaged_at
-        if self.job.status == Job.Status.FINISHED:
-            self.job.project.data_last_packaged_at = self.data_last_packaged_at
-            self.job.project.last_package_job = self.job
-            self.job.project.save(
-                update_fields=(
-                    "data_last_packaged_at",
-                    "last_package_job",
-                )
+        self.job.project.data_last_packaged_at = self.data_last_packaged_at
+        self.job.project.last_package_job = self.job
+        self.job.project.save(
+            update_fields=(
+                "data_last_packaged_at",
+                "last_package_job",
             )
+        )
 
-            try:
-                project_id = str(self.job.project.id)
-                package_ids = storage.get_stored_package_ids(project_id)
-                job_ids = [
-                    str(job["id"])
-                    for job in Job.objects.filter(
-                        type=Job.Type.PACKAGE,
-                    )
-                    .exclude(
-                        status__in=(Job.Status.FAILED, Job.Status.FINISHED),
-                    )
-                    .values("id")
-                ]
+        try:
+            project_id = str(self.job.project.id)
+            package_ids = storage.get_stored_package_ids(project_id)
+            job_ids = [
+                str(job["id"])
+                for job in Job.objects.filter(
+                    type=Job.Type.PACKAGE,
+                )
+                .exclude(id=self.job_id)
```

Comment:
Before self.job was excluded with `FINISHED` jobs. Now need to exclude seperately.